### ISSUE TITLE
Incompatibility with REI

### ISSUE THREAD

**Issue by vlad1m** [mlus-asuka/Thirst-Remade#8]
In the latest version of the mod it does not display in REI the cooking recipes for purifying  water bottles.

**Comment by mlus-asuka** [comment]
Could it have been worked before? I don't think I did anything wrong. the recipe is written in Json, and it works well on JEI.

**Comment by vlad1m** [comment]
It worked with several different versions of REI when the mod was on version 1.0.2, but since version 1.0.3 it stopped working.

**Comment by mlus-asuka** [comment]
obviously, there are something wrong with REI. And I removed codes about JEI plugin since 1.0.3 that's one of the reasons. the plugin is not necessary for JEI, but by REI, funny.
